alma/alma-8.x/common/install_utils.sh

```diff
@@ -7,6 +7,10 @@ yum install -y https://repo.almalinux.org/almalinux/8.6/BaseOS/x86_64/os/Package
     https://repo.almalinux.org/almalinux/8.6/BaseOS/x86_64/os/Packages/kernel-headers-${KERNEL}.rpm \
     https://repo.almalinux.org/almalinux/8.6/BaseOS/x86_64/os/Packages/kernel-modules-extra-${KERNEL}.rpm
 
+# Install Python 3.8
+yum install -y python3.8
+ln -fs /usr/bin/python3.8 /usr/bin/python3
```

Comment:
Have you had a chance to verify waagent behavior and IPoIB?

Reply:
I have an image up but I am not sure how to test these out? Any recommendations?